spec/spec_helper.rb

```diff
@@ -1,47 +1,54 @@
 # frozen_string_literal: true
 
 require 'simplecov'
-SimpleCov.start 'rails'
-SimpleCov.start do
+SimpleCov.start 'rails' do
   add_filter '/config/'
   add_filter '/spec/'
   refuse_coverage_drop
 end
 
 ENV['RAILS_ENV'] ||= 'test'
-require 'spec_helper'
 require File.expand_path('../../config/environment', __FILE__)
 require 'rspec/rails'
 require 'rack_session_access/capybara'
-require 'factory_girl_rails'
-require 'umts-custom-matchers'
 
 ActiveRecord::Migration.maintain_test_schema!
 
 RSpec.configure do |config|
   config.infer_spec_type_from_file_location!
   config.use_transactional_fixtures = true
-  config.before :all do
-    FactoryGirl.reload
-  end
-  config.include FactoryGirl::Syntax::Methods
+
+  config.include FactoryBot::Syntax::Methods
   config.include UmtsCustomMatchers
+
   config.expect_with :rspec do |expectations|
     expectations.include_chain_clauses_in_custom_matcher_descriptions = true
   end
   config.mock_with :rspec do |mocks|
     mocks.verify_partial_doubles = true
   end
+
+  config.before :all do
+    FactoryBot.reload
+  end
 end
 
 def when_current_user_is(user)
-  session[:user_id] = case user
-                      when User
-                        user
-                      when :whoever
-                        create :user
-                      end.id
+  current_user = case user
+                 when User
+                   user
+                 when :whoever
+                   create :user
+                 else
+                   raise ArgumentError
+                 end
+  if defined? page #Capybara
+    page.set_rack_session user_id: current_user.id
+  else #Request specs
+    session[:user_id] = current_user.id
+  end
 end
+alias set_current_user when_current_user_is
```

Comment:
Is there any reason to keep the `set_current_user` alias?

Reply:
Eh, I kinda liked the way it sounded grammar-wise in some of the places I found it?

Reply:
I suppose it doesn't hurt to keep both, but it could cause minor confusion for new developers down the road.

Reply:
I think it's fine. The use of `alias` makes it (fairly) clear that these are the same method. Rather than before where you had to use one or the other depending on which type of test you were writing.